docs/UserGuide.md

```diff
@@ -58,121 +68,131 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+<img src="images/user-guide/projectsView.png" width="300" />
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+#### Statistics View
 
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+<img src="images/user-guide/statisticsView.png" width="300" />
 
-### Deleting a person : `delete`
+### Timer
 
-Deletes the specified person from the address book.
+Start and end a timer to record the duration of a project.
 
-Format: `delete INDEX`
+#### Start Timer: `start`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+Format: `start PROJECT_ID`
 
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* Starts the timer with `PROJECT_ID` specified
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
 
-### Clearing all entries : `clear`
+Example: `start 1`
 
-Clears all entries from the address book.
+Result: Start timer with project id 1.
 
-Format: `clear`
+#### End Timer: `end`
 
-### Exiting the program : `exit`
+Format: `end PROJECT_ID`
 
-Exits the program.
+* Ends the timer with `PROJECT_ID` specified
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
 
-Format: `exit`
+Example: `end 1`
 
-### Saving the data
+Result: End timer with project id 1.
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+### Exiting the program : /`exit`
 
-### Archiving data files `[coming in v2.0]`
+Format: `/exit`
 
-_{explain the feature here}_
+Result: Exits the program.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Momentum home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
```

Comment:
Should we separate format and examples into two columns?

Reply:
Not sure, but if we add a bunch examples, then a table would be like 1 line | a lot of lines